pysoa/client/client.py

```diff
@@ -570,6 +570,90 @@ def call_jobs_parallel(
 
         return responses
 
+    # Methods used to send a request in a non-blocking manner and then later block for a response as a separate step
+
+    def send_request(
```

Comment:
This method is not new. It was moved from the bottom of the class to here, above all the private methods.